README.md

```diff
@@ -52,16 +52,25 @@ $ npm install jasmine-enzyme --save-dev
 
 ## Setup
 
-###### Jest
-If you are using [jest](https://facebook.github.io/jest/), the simplest setup is to use jests `setupTestFrameworkScriptFile` config. Add this to your `package.json`
+### Jest
+If you are using [jest](https://facebook.github.io/jest/), the simplest setup
+is to use jest's `setupTestFrameworkScriptFile` config. You'll also need to
+tell jest to unmock `react`, `enzyme`, and `enzyme-jest`.
```

Comment:
`enzyme-jest` ? do you mean `jasmine-enzyme` ?

Also. I've never experienced needing jasmine-enzyme in the unmocked module paths. From having a slight understanding of jest's internals, I don't think it would ever be necassary. Could you try removing jasmine-enzyme from your unmockedModulePathPatterns and see if it fails?